static/js/publisher/metrics/colorScale.js

```diff
@@ -0,0 +1,23 @@
+export default function(steps) {
+  const start = [ 255, 232, 200 ];
+  const end = [ 226, 74, 51 ];
```

Comment:
If you wanted the same colors as the map:
```
maxColor = rgb(8, 64, 129)
minColor = rgb(29, 245, 223)
```

Reply:
this is for the activeDevices, but good to know!

Reply:
It's not ideal that we hardcode colours in the code.
If we can't move these into CSS because of functionality maybe we can later have some file were we define colour variables/constants or something...

Reply:
I have a config file already, will move them there